parcels/application_kernels/advectiondiffusion.py

```diff
@@ -24,7 +26,7 @@ def AdvectionDiffusionM1(particle, fieldset, time):  # pragma: no cover
     The Wiener increment `dW` is normally distributed with zero
     mean and a standard deviation of sqrt(dt).
     """
-    dt = particle.dt / np.timedelta64(1, "s")  # noqa TODO improve API for converting dt to seconds
```

Comment:
Don’t we want to keep the reminder note that we may want to consider the API for dt?

Reply:
Yes, I've added them back.

They were automatically removed by [Ruff](https://docs.astral.sh/ruff/rules/unused-noqa/) (if it was formatted like `# noqa # my comment`, Ruff would have kept `# my comment`).